libs/ng-morph/utils/helpers/get-declaration-getter.ts

```diff
@@ -11,8 +11,11 @@ export function getDeclarationGetter<
     pattern: Pattern,
     query?: Query<Omit<Structure, 'kind'>>
   ): Declaration[] {
+
     return getFn(pattern).filter((declaration) =>
+      // @ts-ignore
       matchQuery(declaration.getStructure(), query)
```

Comment:
what is the issue here? Maybe we could solve it more robust way

Reply:
As far as I understand, with the update of TypeScript version, the type checking has been improved, and now `getStructure` doesn't want to be passed to the `editor` function. Unfortunately, I couldn't fix that.

<img width="1083" alt="image" src="https://github.com/Tinkoff/ng-morph/assets/20700969/90532966-d3f7-4030-84fc-f418801d0f51">

Reply:
all right, please let's add a TODO comment and we'll solve it later
`TODO: refactor it to support new typings`

Reply:
the comment has been added